### PR TITLE
chore: better setup

### PR DESCRIPTION
**Pull request by iiAbady**
## **Describe what does this pull request do?**
This pull request is intended to further improve the setup of the codebase by having a simple pull request template and an auto assigner for pull request.

### CODE REVIEW

**Review by SufyanCS — APPROVED** [review]

File: public/favicon.ico
Comment (adeeb-samad) [inline public/favicon.ico:1]: what are these?

File: public/next.svg
Comment (adeeb-samad) [inline public/next.svg:1]: images will be put in ~/public/images/...

**Review by adeeb-samad — APPROVED** [review]
Done

File: public/next.svg
Comment (iiAbady) [reply public/next.svg:1]: It's not specifically an image but rather the icon of the website.

**Pull request merged**